code/modules/mob/living/living_verbs.dm

```diff
@@ -61,9 +61,7 @@
 				C.last_special = world.time + 100
 				C.visible_message("<span class='danger'>[C] attempts to unbuckle themself!</span>",\
 				"<span class='warning'> You attempt to unbuckle yourself. (This will take around 2 minutes and you need to stand still)</span>")
-				if(do_after(C, 1200, FALSE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
I have to be honest and say that having a busy indicator for minutes while (buckle)cuffed is a punch to the already low chances of uncuffing oneself. Feel free to disapprove it anyway.

Reply:
You shouldn't change functionality here